packages/components/src/components/loading/loading.scss

```diff
@@ -58,3 +58,177 @@
         }
     }
 }
+
+.loading-dtrader-v2 {
+    &__header {
+        display: flex;
+        justify-content: space-between;
+        padding: var(--core-spacing-300) var(--semantic-spacing-general-md);
+    }
+    &__trade,
+    &__positions,
+    &__contract-details {
+        display: flex;
+        flex-direction: column;
+        height: 100%;
+    }
+    &__trade {
+        background-color: var(--core-color-opacity-black-75);
+        padding: 0 var(--core-spacing-400);
+        gap: var(--semantic-spacing-general-sm);
+        position: relative;
+        overflow-x: hidden;
+
+        .skeleton-box,
+        .skeleton-box__trade-types,
+        .skeleton-box__trade-params,
+        .skeleton-box__chart {
+            display: flex;
+            flex-direction: column;
+            border-radius: var(--semantic-borderRadius-md);
+            background-color: var(--component-modal-bg);
+
+            &__trade-types {
+                position: relative;
+                inset-inline-start: calc(-1 * var(--semantic-spacing-general-sm));
+                width: 100vw;
+                flex-direction: row;
+                flex-wrap: nowrap;
+                border-radius: unset;
+                padding: var(--semantic-spacing-general-sm) var(--semantic-spacing-general-md);
+                gap: var(--semantic-spacing-general-sm);
+
+                .skeleton {
+                    flex-shrink: 0;
+                }
+            }
+            &__market {
+                padding: 0 var(--semantic-spacing-general-sm);
+            }
+            &__trade-params {
+                padding: var(--semantic-spacing-general-md);
+                gap: var(--semantic-spacing-general-md);
+
+                &-row {
+                    display: flex;
+                    justify-content: space-between;
+                    align-items: center;
+                }
+                &-column {
+                    display: flex;
+                    flex-direction: column;
+                    gap: var(--semantic-spacing-general-sm);
+                }
+            }
+            &__chart {
+                min-height: 400px;
+                padding: var(--semantic-spacing-general-sm);
+            }
+        }
+    }
+    &__positions {
+        width: 100%;
+        background-color: var(--core-color-solid-slate-75);
+
+        .skeleton-box {
+            &__filters,
+            &__total-pnl-or-date {
+                display: flex;
+                gap: var(--semantic-spacing-gap-md);
+                margin: var(--semantic-spacing-gap-md) 0;
+                padding: 0 var(--semantic-spacing-gap-md);
+            }
+            &__tabs {
+                display: flex;
+                background-color: var(--component-modal-bg);
+                width: 100%;
+                justify-content: space-around;
+                padding: 1.3rem 0;
```

Comment:
Do we have quill token for it? Please, ignore if irrelevant

Reply:
no, we don't)